<ui version="4.0" >
 <class>QDialogPixmap</class>
 <widget class="QDialog" name="QDialogPixmap" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Text properties</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth" >
        <number>3</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <property name="sizeConstraint" >
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="spinX" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>-61.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>61.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="spinY" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>-61.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>61.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3" >
            <property name="fieldGrowthPolicy" >
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Image:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLineEdit" name="lineEdit" />
            </item>
            <item row="1" column="1" >
             <widget class="QPushButton" name="btnImage" >
              <property name="text" >
               <string>Load image...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCDView" name="cdView" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>2</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize" >
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="interactive" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCDView</class>
   <extends>QGraphicsView</extends>
   <header>qcdview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinX</tabstop>
  <tabstop>spinY</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDialogPixmap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>258</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDialogPixmap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>326</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
